mgmtsystem/__manifest__.py

```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
@sbidoul 
i just checked the 9.0 __openerp__.py
and it had still board and document_page in the dependencies even though readme tells its not depending on document_page anymore.

I testet the base module and the other that ive ported, they all install without board, but some have document_page as a dependency.

So its save to remove board and document_page from dependencies in the core module.